src/dbnode/generated-source-files.mk

```diff
@@ -22,29 +22,23 @@ install-m3x-repo: install-glide
 	)
 
 # Generation rule for all generated types
-.PHONY: genny-all-dbnode
-genny-all-dbnode: genny-map-all-dbnode genny-arraypool-all-dbnode genny-leakcheckpool-all-dbnode
-
-# Tests that all currently generated types match their contents if they were regenerated
-.PHONY: test-genny-all-dbnode
-test-genny-all-dbnode: genny-all-dbnode
-	@test "$(shell git diff --shortstat 2>/dev/null)" = "" || (git diff --no-color && echo "Check git status, there are dirty files" && exit 1)
-	@test "$(shell git status --porcelain 2>/dev/null | grep "^??")" = "" || (git status --porcelain && echo "Check git status, there are untracked files" && exit 1)
+.PHONY: genny-all
+genny-all: genny-map-all genny-arraypool-all genny-leakcheckpool-all
 
 # Map generation rule for all generated maps
-.PHONY: genny-map-all-dbnode
-genny-map-all-dbnode:                         \
-	genny-map-client-received-blocks-dbnode     \
-	genny-map-storage-block-retriever-dbnode    \
-	genny-map-storage-bootstrap-result-dbnode   \
-	genny-map-storage-dbnode                    \
-	genny-map-storage-namespace-metadata-dbnode \
-	genny-map-storage-repair-dbnode             \
-	genny-map-storage-index-results-dbnode
+.PHONY: genny-map-all
```

Comment:
Nice! I was hating this fragility/verbosity/

Reply:
For sure. :)